docs/src/components/pages/org/org.html

```diff
@@ -0,0 +1,2 @@
+{% render '@page-header--with-icon' %}
```

Comment:
As a second argument to `render`, you can pass a JS object which will override properties in the component's `context`.